.idea/vcs.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="IssueNavigationConfiguration">
+    <option name="links">
+      <list>
+        <IssueNavigationLink>
+          <option name="issueRegexp" value="#(\d+)" />
+          <option name="linkRegexp" value="https://github.com/mobxjs/mobx/issues/$1" />
```

Comment:
Sometimes the number can be a PR, so it feels too strange linking this to issue always. Not sure if there is a better way, I don't use Idea products.

Reply:
Issues and PRs have single index in GitHub
For example current PR available [by this link](https://github.com/mobxjs/mobx/issues/2335)

Reply:
Alright, did not know about that, cool :)